va/validation-authority.go

```diff
@@ -208,6 +237,15 @@ func (va ValidationAuthorityImpl) validateDvsni(identifier core.AcmeIdentifier,
 	z := sha256.Sum256(RS)
 	zName := fmt.Sprintf("%064x.acme.invalid", z)
 
+	// Check for resolver SERVFAIL for A/AAAA records
```

Comment:
The reason I originally added this back in was so that we could distinguish between a `SERVFAIL` and `NXDOMAIN` since the TLS & HTTP stacks return a `net.DNSError` that will only tell us if there was a time out or an underlying transport error.

Given the discussion yesterday about (possibly) performing validation against all A/AAAA records for a domain we may want to keep these checks (and underlying method) so that we can bypass the resolution the `http` lib is doing and manually pick an address to validate against/iterate through all addresses and do validation (although this is work for another PR).